src/components/icons/MotoScout24Logo.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { createIcon } from '@chakra-ui/react';
+
+export const MotoScout24Logo = createIcon({
```

Comment:
This seems to be the app version of the logo, I wonder 
1. If we should name the variation somehow
2. If we should use the same variation on the error page

![image](https://user-images.githubusercontent.com/3371760/199214271-a2a6c6e4-7d6b-4fbf-8f74-687411b86295.png)

Reply:
for reference: https://scout24-ch.frontify.com/document/8?brand20=AutoScout24%2CMotoScout24

Reply:
yeah I am waiting for the app version of AutoScout from Enrique.. there is no svg in frontify 🙄

Reply:
done ✅